Unix/wsman/wsmanerrorhandling.h

```diff
@@ -308,5 +309,4 @@ const Probable_Cause_Data* GetWsmanCimError(Error_Types type);
 
 END_EXTERNC
 
-
-
+#endif
```

Comment:
By convention, since this `#endif` is hundreds of lines later, I prefer something like:

```#endif // !defined(_wsmanerrorhandling_h)```

This makes it easier to see what's what in the header file.